### ISSUE TITLE
Failed to define function cl-old-struct-compat-mode

### ISSUE THREAD

**Issue by hajovonta** [portacle/portacle#88]
I upgraded to the newest portacle today (Windows). During initializing, I get this error in Emacs:
Autoloading file c:/Users/user/portacle/win/emacs/share/emacs/25.2/lisp/emacs-lisp/cl-lib.elc failed to define function cl-old-struct-compat-mode

This is before starting SBCL.

**Comment by Shinmera** [comment]
Did you install it on top of a previous installation? If so, please remove the `portacle/win/emacs/share/emacs/25.2` folder.

**Comment by hajovonta** [comment]
Yes, I did install it on top of a previous install. Now I deleted the folder that you suggested, portacle won't start. I see a console window popping up momentarily, then nothing. (Windows 10)

**Comment by Shinmera** [comment]
Try deleting everything and just copying over projects/ and config/ if you changed those. A clean install worked for me.
